lib/resourcebuilder/podspec.go

```diff
@@ -45,3 +45,78 @@ func updatePodSpecWithProxy(podSpec *corev1.PodSpec, containerNames []string, ht
 	return nil
 
 }
+
+// updatePodSpecWithInternalLoadBalancerKubeService mutates the input podspec by setting the KUBERNETES_SERVICE_HOST to the internal
+// loadbalancer endpoint and the KUBERNETES_SERVICE_PORT to the specified port
+func updatePodSpecWithInternalLoadBalancerKubeService(podSpec *corev1.PodSpec, containerNames []string, internalLoadBalancerHost, internalLoadBalancerPort string) error {
```

Comment:
can we add a comment to [the deployment itself][1], so folks reading that know that we may be fiddling with some fields dynamically?

[1]: https://github.com/openshift/cluster-version-operator/blob/6132bc33a76762dd1071eb049e308a4df75673ee/install/0000_00_cluster-version-operator_03_deployment.yaml

Reply:
> can we add a comment to the deployment itself...

95c6c57 has this; marking the thread resolved.